### PR TITLE
A0-1124 Fix so that DRIP_AMOUNT bigger than 9007 are supported

### PR DESCRIPTION
**Pull request by Marcin-Radecki**
Tested locally:

image

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,32 +16,36 @@ yarn dev:backend
...
-RPC_ENDPOINT="https://westend-rpc.parity.io/"
+RPC_ENDPOINT="http://localhost:9933"
+GOOGLE_CAPTCHA_PRIVATE=6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe
+NETWORK_UNIT="DZERO"
+COOLDOWN=100
+BACKEND_URL=http://localhost:5555
+DRIP_AMOUNT=150
 ```
 
+Google captch secret key is a public [well-known test key](https://developers.google.com/recaptcha/docs/faq#id-like-to-run-automated-tests-with-recaptcha.-what-should-i-do),
```
Comment (piotrMocz) [inline README.md]: Typo: captch**a** :D

**Review by piotrMocz — APPROVED** [review]
Looks good! Btw, have you tested what happens when the `amount` is not parsable into a number?

**Review by DamianStraszak — APPROVED** [review]
I'm kind of shocked that the fix is so local and simple, I hope that it will work :P

File: README.md (hunk 1 above)
Comment (Marcin-Radecki) [reply README.md]: thanks, done

**Pull request merged**